package/Shaders/Lighting.hlsl

```diff
@@ -1696,7 +1696,12 @@ PS_OUTPUT main(PS_INPUT input, bool frontFace
 	float3 positionMSSkylight = input.WorldPosition.xyz;
 #		endif
 
+#		if defined(DEFERRED)
 	sh2 skylightingSH = Skylighting::sample(skylightingSettings, SkylightingProbeArray, positionMSSkylight, worldSpaceNormal);
+#		else
+	sh2 skylightingSH = (PixelShaderDescriptor & _InWorld || !FrameParams.y) ? Skylighting::sample(skylightingSettings, SkylightingProbeArray, positionMSSkylight, worldSpaceNormal) : float4(sqrt(4.0 * shPI), 0, 0, 0);
```

Comment:
What's this doing?

Reply:
neutralising skylighting if it's not part of the world scenegraph